CHANGES.rst

```diff
@@ -12,7 +12,10 @@ Changes
 
 Changes:
 --------
-- No change.
+- Support `CWL` definition for ``cwltool:CUDARequirement`` to request the use of a GPU, including support for using
+  Docker with a GPU.
```

Comment:
Add note about https://github.com/crim-ca/weaver/issues/104 resolution (see other references in changes for example).